docs/en/sql-reference/table-functions/url.md

```diff
@@ -41,7 +41,7 @@ INSERT INTO FUNCTION url('http://127.0.0.1:8123/?query=INSERT+INTO+test_table+FO
 SELECT * FROM test_table;
 ```
 
-## Globs in URL {globs-in-url}
+## Globs in URL
```

Comment:
It would be more correct to fix it as `## Globs in URL {#globs-in-url}` -- same as it is done in many other places (example https://github.com/ClickHouse/ClickHouse/blob/master/docs/en/engines/table-engines/integrations/s3.md). It allows to get a link to specific place on the page.